data/frameProcessor/src/FrameMem.cpp

```diff
@@ -17,10 +17,9 @@
 * @ret negative is failure.
 */
 template<>
-int64_t FrameMem<float>::loadFromH5(std::string filename, std::string dataset, int frameNo, std::string* outError)
+int64_t FrameMem<float>::loadFromH5(std::string filename, std::string dataset, int frameNo)
```

Comment:
A lot cleaner now with the debugger.

Reply:
yeah it's better but there is a dependency on log4cxx, but that's standard at Dmd.